srcpkgs/python3-textual/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-textual'
+pkgname=python3-textual
+version=0.45.1
+revision=1
+build_style=python3-pep517
+make_check_args="-k not((snapshot)or(markdown)or(feature)or(language))"
+hostmakedepends="python3-poetry-core"
+depends="python3-rich python3-markdown-it python3-importlib_metadata python3-typing_extensions"
+checkdepends="${depends} python3-pytest-asyncio python3-pytest-xdist"
+short_desc="Modern Text User Interface framework"
+maintainer="classabbyamp <void@placeviolette.net>"
```

Comment:
why'd I get put here? I closed that PR because I wasn't using this

Reply:
Oh no! That's a copy-paste error. I forgot to update the field. Will fix in next update.

Reply:
i'll just do it now